lib/identity/client.js

```diff
@@ -141,6 +139,35 @@ class Client extends PartialClient {
     const resultQuery = await this.storage.search(request)
     return resultQuery
   }
+  async updateSecret(secret) {
```

Comment:
This should take in the old secret to update, and the new value to update it with